docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -734,6 +734,29 @@ class SharedService {
 <2> `@Lock(Lock.Type.READ)` overrides the value specified at class level. It means that any number of clients can invoke the method concurrently, unless the bean instance is locked by `@Lock(Lock.Type.WRITE)`. 
 <3> You can also specify the "wait time". If it's not possible to acquire the lock in the given time a `LockException` is thrown.
 
+=== Repeatable interceptor bindings
+
+Quarkus supports `@Repeatable` interceptor binding annotations, if they are declared on methods.
+For example:
+
+[source,java]
+----
+@ApplicationScoped
+class AllEncompassingService {
+  @Cached(cacheName = "foo") // <1>
+  @Cached(cacheName = "bar")
```

Comment:
There's another reason why `@Cached` (even the imaginary version) might not be the best pick for that example: it doesn't really make sense to repeat an annotation aimed at retrieving data from a cache because having multiple caches on a single method would raise a lot of questions. That's probably why most (if not all) annotations-based caching API do not allow this.

Reply:
Yea I know it's stupid. The other example too. If you have a better one, I'm all ears :-)